### DISCUSSION TITLE
Ignore query parameters in ISR with fallback: true

### DISCUSSION THREAD

**Discussion by hatsumatsu** [vercel/next.js#24091] [Help]
Using ISR with `fallback: true` I noticed that query parameters added to referer links (like Facebook's ?fbclick=xxxx) trigger SSR on my dynamic routes. (At least that is what vercel's log suggest)

Let's say I have a route `/pages/path/[slug].js` and use `{ revalidate: 60, fallback:true }` the following URLs seem to generate three static pages while only one is needed:

`/path/abc`
`/path/abc?fbclick=123`
`/path/abc?fbclick=456`

Is there a way to ignore specific query parameters since I have no control over them? Would be great to save some computation time on hosts like vercel.

**Comment by hatsumatsu** [comment]
Ideally there would be an allowlist and ignorelist for query parameters either on a global or per-page basis.

An allowlist would be helpful since every third-party service linking to the site could add tracking parameters (social media, newsletter tools, affiliate links etc....)

**Comment by hatsumatsu** [comment]
Anyone else having this issue and a potential solution? Dynamic query parameters from third parties are really pushing my site towards vercel's computation limit 😱

**Reply by gabrielpra1** [reply]
Also having trouble with this. Did you find a solution @hatsumatsu?

**Reply by hatsumatsu** [reply]
@gabrielpra1 Unfortunately not. Would love to bump this up somehow or at least to know if this is on next's roadmap .

**Comment by inacior** [comment]
I have opened a issue https://github.com/vercel/next.js/issues/27638 about this, waiting help

**Comment by hatsumatsu** [comment]
See https://github.com/vercel/next.js/issues/27638#issuecomment-930463123
